lib/internal/http2/compat.js

```diff
@@ -223,43 +194,48 @@ class Http2ServerRequest extends Readable {
   }
 
   get method() {
-    var stream = this[kStream];
-    if (!stream) return;
-    var headers = stream[kHeaders];
-    return headers.get(constants.HTTP2_HEADER_METHOD);
+    var headers = this[kHeaders];
+    if (!headers)
```

Comment:
I believe core is generally moving towards explicitly checking against `=== undefined` with these types of if statement